cluster-autoscaler/cloudprovider/azure/azure_template.go

```diff
@@ -204,6 +389,37 @@ func extractTaintsFromScaleSet(tags map[string]*string) []apiv1.Taint {
 	return taints
 }
 
+// Example of a valid taints string, is the same argument to kubelet's `--register-with-taints`
+// "dedicated=foo:NoSchedule,group=bar:NoExecute,app=fizz:PreferNoSchedule"
+func extractTaintsFromSpecString(taintsString string) []apiv1.Taint {
```

Comment:
This is actually not being used at the point of this commit. May be removed from this PR.